<launch>

<node name="aitsmc_heading" pkg="usv_ros" type="aitsmc_heading" >
  <param name = "k_u" value = "0.5" />
	<param name = "k_psi" value = "0.75" />
	<param name = "kmin_u" value = "0.01" />
	<param name = "kmin_psi" value = "0.01" />
	<param name = "k2_u" value = "0.01" />
   <param name = "k2_psi" value = "0.001" />
	<param name = "mu_u" value = "0.05" />
	<param name = "mu_psi" value = "0.02" />
	<param name = "alpha_u" value = "1.0" />
	<param name = "alpha_psi" value = "0.750" />
	<param name = "beta_psi" value = "0.750" />
    <param name = "q_u" value = "3" />
	<param name = "q_psi" value = "3" />
    <param name = "p_u" value = "5" />
	<param name = "p_psi" value = "5" />
</node>

</launch>